modules/managers/galasa-managers-parent/galasa-managers-imstm-parent/dev.galasa.imstm.manager/src/main/java/dev/galasa/imstm/ImsTerminal.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright contributors to the Galasa project
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package dev.galasa.imstm;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import dev.galasa.framework.spi.ValidAnnotatedFields;
+
+/**
+ * A zOS 3270 Terminal for use with an IMS TM System that has access to the default IMS screens
+ * 
+ * <p>
```

Comment:
FYI: These <p> markers do nothing if you already left a blank line. A blank line in Javadoc is treated as a new paragraph.